docs/book/index.rst

```diff
@@ -16,18 +16,17 @@ Introduction aims to describe the philosophy of Sylius. It will also teach you a
 
 .. include:: /book/introduction/map.rst.inc
 
-Installation
-------------
+Setup
+-----
 
-The installation chapter is of course a comprehensive guide to installing Sylius on your machine, but it also provides
+The setup chapter is of course a comprehensive guide to installing Sylius on your machine, but it also provides
 a general instruction on upgrading Sylius in your project.
 
 .. toctree::
-    :hidden:
-
-    installation/index
+    :titlesonly:
```

Comment:
is it necessary?

Reply:
Without it, we'd get sections like "Before installation" and "Learn more" listed.